tests/mocks/services.py

```diff
@@ -1173,6 +1174,9 @@ def __init__(
         available: bool,
         location_auth: LocationAuth,
         robot_part_history: List[RobotPartHistoryEntry],
+        fragment_history: List[FragmentHistoryEntry],
+        page_token: str,
```

Comment:
in the proto these are optional - should they be optional here as well ?

Reply:
I actually think we shouldn't make this a default part of the class at all. Currently we're setting them to our sentinel values when we create the service, and then passing those same sentinel values when we call the `get_fragment_history` method in the test, and then confirming that they're set to the sentinel value. But, because there's two places where the setting happens, we don't have confidence that the test actually did anything. 

What I'd suggest instead is just removing these as arguments or set values in the constructor, such that they don't exist in general. Then, we can leave everything else as it is such that the `GetFragmentHistory` method sets these values, and then we can meaningfully confirm that they're properly set in the `test_get_fragment_history` test.

Reply:
I got rid of page_token and page_limit but I kept fragment_history because that was the return value - does that make sense ? Apologies if I'm misunderstanding.